CONTRIBUTING.adoc

```diff
@@ -65,17 +66,21 @@ $ brew install git
 ----
 
 Then, follow the instructions to install https://docs.docker.com/docker-for-mac/install/[`docker`].
+Use the link here to download https://download.docker.com/mac/stable/21090/Docker.dmg[version 17.09.1].
```

Comment:
s/link here to/link to (for all of the following as well)

Reply:
linking to specific versions is just setting us up for bit-rotting, which is probably why the instructions weren't specific in the first place.  I'd suggest keeping it that way unless it's particularly difficult to locate the image from the linked instructions.